docs/source/getting_started.rst

```diff
@@ -192,14 +196,19 @@ When you stop a scan by using the ``scan pause`` command, you can restart that s
 
 1. Make sure that you have the scan job identifier for the currently running scan. To obtain the scan job identifier, see the information in `Showing Scan Job Results for an Active Scan`_.
 
-2. Enter the command to pause the scan. For example, if the scan job identifier is ``1``, you would enter the following command:
-
-  ::
+2. Enter the command to pause the scan job. For example, if the scan job identifier is ``1``, you would enter the following command::
 
     # qpc scan pause --id 1
 
-3. When you are ready to start the scan again, enter the command to restart the scan. For example, to restart scan ``1``, you would enter the following command:
-
-  ::
+3. When you are ready to start the scan job again, enter the command to restart the scan. For example, to restart scan ``1``, you would enter the following command::
 
     # qpc scan restart --id 1
+
+
+Logging out of the Quipucords Server
+------------------------------------
+When you log in to the server, the command retrieves a token that is used for authentication with subsequent command line interface commands. That token expires daily. In addition, the token is removed when you log out of the server.
```

Comment:
whatever we decide about "into" vs. "in to" should be carried over to here as well.

Reply:
yeah i don't know the rules. Let Jamie do Jamie.